lib/api/graph.js

```diff
@@ -43,8 +44,63 @@ function snapshot(query, from, to, eventQuery, callback){
     client.request("GET", "/graph/snapshot", params, callback);
 }
 
+/*section: graph
+ *comment: create an embed graph of a metric query
+ *params:
+ *  graph_json: The request array to pass create in the embed
+ *  timeframe: (1_hour, 4_hours, 1_day, 2_days, and 1_week)
+ *  size: (small, medium, large, xlarge)
+ *  legend: yes or no
+ *  title: The title of the embed
+ *  callback: function(err, res)
+ *example: |
+ *  ```javascript
+ *  var dogapi = require("dogapi");
+ *  var options = {
+ *    api_key: "api_key",
+ *    app_key: "app_key"
+ *  };
+ *  dogapi.initialize(options);
+ *  var query = "system.cpu.idle{*}";
+ *  var graph_json = {
+ *         "viz": "timeseries",
+ *         "requests": [
+ *           {
+ *             "q": "system.cpu.idle{*}",
+ *             "aggregator": "avg",
+ *             "conditional_formats": [],
+ *             "type": "area"
+ *           }
+ *         ]
+ *       }
+ *  var timeframe = '1_hour';
+ *  var size = 'xlarge';
+ *  var legend = 'yes';
+ *  var title = 'my awesome embed';
+ *  dogapi.graph.embed(graph_json, timeframe, size, legend, title, function(err, res){
+ *    console.dir(res);
+ *  });
+ *  ```
+ */
+function createEmbed(graph_json, timeframe, size, legend, title, callback){
+    var lastArgument = arguments.length -1;
+
+    var params = {
+        body: 'graph_json=' + JSON.stringify(graph_json) +
+            '&timeframe=' + timeframe +
+            '&size=' + size +
+            '&legend=' + legend +
+            '&title=' + title
+        ,
+        contentType: 'application/x-www-form-urlencoded'
+    };
+
+    client.request("POST", "/graph/embed", params, callback);
+}
+
 module.exports = {
     snapshot: snapshot,
+    createEmbed: createEmbed,
     getUsage: function(){
         return [
             "  dogapi graph snapshot <query> <from> <to> [--events <event-query>]"
```

Comment:
Still missing usage string here e.g. `dogapi graph  create_embed --graph_json <graph_json> --timeframe <timeframe> --size <size> --legend <legend> --title <title>`

I know it is a little tedious to document this stuff... if I could go back in time...